apps/crn-server/test/data-providers/research-outputs.data-provider.test.ts

```diff
@@ -978,6 +1001,39 @@ describe('ResearchOutputs data provider', () => {
         ).resolves.not.toThrow();
       });
 
+      test('Should publish a draft research-output', async () => {
```

Comment:
why dont you create a separaete describe block where you test:
1. Sending the status option as Published
2. Sending the status option as Draft
3. Not sending it at all

In 3) we should show that the request to change the status does not happen